packages/react-native-reanimated/src/ConfigHelper.ts

```diff
@@ -58,7 +61,9 @@ export function configureReanimatedLogger(config: LoggerConfig) {
   // Update the configuration object in the React runtime
   updateLoggerConfig(config);
   // Register the updated configuration in the UI runtime
-  executeOnUIRuntimeSync(updateLoggerConfig)(config);
+  if (!SHOULD_BE_USE_WEB) {
+    executeOnUIRuntimeSync(updateLoggerConfig)(config);
+  }
```

Comment:
I tried to add an interactive playground and was getting crash in there. After that I decided that interactive example makes no much sense in this case as logs on web are logged to the console and many people may assume that this example because they won't see any warnings/errors until they open dev tools.

At least, thanks to this I was able to find the issue and fix it.